### PR TITLE
prepare for ietf version

### PR DESCRIPTION
**Pull request by suhasHere**

### CODE REVIEW

**Comment by fluffy** [comment]
can this be closed ?

**Comment by suhasHere** [comment]
This needs to be merged to get the version that is in the data tracker.

**Comment by suhasHere** [comment]
@ianswett , this PR needs to be merged since the current version on the main branch doesn't reflect what is submitted to the IETF .

**Review by ianswett — APPROVED** [review]

**Review by fluffy — APPROVED** [review]
FWIW ... I build, diffed, and looks good to me.

**Comment by kixelated** [comment]
```
git rm draft-lcurley-moq-transport.md
git commit
```

Otherwise there will be two copies of the draft in the repo. If you rm in this PR, then the diff will show up on GitHub.

**Comment by kixelated** [comment]
I pushed the delete.

File: draft-ietf-moq-transport.md [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
 title: "Media over QUIC Transport"
 abbrev: moq-transport
-docname: draft-lcurley-moq-transport-latest
+docname: draft-ietf-moq-transport-00
 date: {DATE}
-category: info
+category: std 
 
 ipr: trust200902
 area: General
```
Comment (kixelated) [inline draft-ietf-moq-transport.md:9]: art

**Review by wilaw — APPROVED** [review]

File: draft-ietf-moq-transport.md (hunk 1 above)
Comment (ianswett) [inline draft-ietf-moq-transport.md:9]:
```suggestion
area: Applications and Real-Time
```

**Pull request merged**